hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -387,6 +388,19 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 				ObservedGeneration: nodePool.Generation,
 			})
 		}
+	} else if nodePool.Spec.Platform.Type == hyperv1.PowerVSPlatform {
+		powervsImage, _, err := getPowerVSImage(hcluster.Spec.Platform.PowerVS.Region, releaseImage)
+		if err != nil {
+			setStatusCondition(&nodePool.Status.Conditions, hyperv1.NodePoolCondition{
+				Type:               hyperv1.NodePoolValidAMIConditionType,
```

Comment:
Does the AMI term has a meaning in PowerVS? Or this just copied from AWS?

Reply:
I just influenced this condition with the previous code where we used this condition even outside the AWS context and seems like it got removed via https://github.com/openshift/hypershift/commit/0c5de07ce205e919efca89bcaeb4aed8ec5bba59. I will introduce the PowerVS specific condition for the Image here!